docs/content/users/quickstart.md

```diff
@@ -16,6 +16,11 @@ Once DDEV is installed, you can quickly spin up new projects:
 
 DDEV comes ready to work with any PHP project, and has deeper support for several common PHP platforms and content management systems.
 
+!!!tip
+    If you're looking for place to test all these examples and you don't have time to install docker or create new workspace then hit the following button.
+
+    [![New DdevSpace](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/ibrah3m/DdevSpace.git)
+
```

Comment:
While I can see value in providing a generic playground area, I think it should be within a `[drud/ddev](https://github.com/drud/ddev)` namespace. This would allow people DDEV-authorized users to manage as appropriate.

Reply:
how to make this happen?

Reply:
@rfay will probably decide if its needed or not, especially if we used the offical domain.

Personally, I think it would be helpful if we had a repo for each supported framework.
That would allow anyone to instantly spin up a Drupal, Laravel, CraftCMS, etc. which would also make it easy to test issues.